src/components/App/AppPages.component.js

```diff
@@ -16,6 +16,7 @@ export const AppPages = () => (
         <Route path="/new" component={NewPage} />
         <Route path="/enrollment/stageEvents" component={StageEventListPage} />
         <Route path="/enrollment/event/edit" component={EnrollmentEventPage} />
+        <Route path="/enrollment/stageEvents" component={StageEventListPage} />
```

Comment:
`<Route path="/enrollment/stageEvents" component={StageEventListPage} />` already exist in the AppPages. Did you mean to add another page?

Reply:
hey @simonadomnisoru, no it was an issue post merge. I will remove it.